git/git-version-control.md

```diff
@@ -170,7 +170,7 @@ be)
 
 - [Interactive Git course](https://try.github.io)
 - [Single page cheatsheet](http://rogerdudler.github.io/git-guide/)
-- [Introduction to Git Concepts (10 min read + diagram)](https://www.intertech.com/Blog/introduction-to-git-concepts/)
+- [Introduction to Git Concepts (10 min read + diagram)](./introduction-to-git-concepts.html)
```

Comment:
to replace

Reply:
How about https://www.freecodecamp.org/news/what-is-git-and-how-to-use-it-c341b049ae61/. ?  There may be better ones, but we do like free code camp.

Reply:
An example of what _not_ to do is:  https://hpckp.org/wp-content/uploads/2020/05/hpckp11_GIT_introduction.pdf

This can be intimidating for someone in their 4th or 5th week of computer science ever.